Modules/_io/winconsoleio.c

```diff
@@ -998,6 +988,11 @@ _io__WindowsConsoleIO_write_impl(winconsoleio *self, Py_buffer *b)
     Py_BEGIN_ALLOW_THREADS
     wlen = MultiByteToWideChar(CP_UTF8, 0, b->buf, len, wbuf, wlen);
     if (wlen) {
+        /* Note that it's possible that this would fail to print big strings
+         * on Windows 7 (before Windows 8 LPC-based pseudo-files were used
+         * for I/O and would be limited to a 64 KiB heap).
+         * As Windows 7 is currently unsupported, that's Ok.
```

Comment:
```suggestion
         * As Windows 7 is no longer supported, that's OK.
```

If we're going to have a comment in here, let's not make it sound like we _might_ bring back support one day.

Reply:
Agreed.